CHANGES.rst

```diff
@@ -267,6 +267,9 @@ astropy.units
 astropy.utils
 ^^^^^^^^^^^^^
 
+- ``astropy.utils.data.download_file`` now provides an `http_headers` keyword to
```

Comment:
This should be under "API change," not "new features," right?

Also, formatting typo:
```suggestion
- ``astropy.utils.data.download_file`` now provides an ``http_headers`` keyword to
```